cli/src/torii/mod.rs

```diff
@@ -34,6 +34,7 @@ pub struct Torii {
     queue: Arc<Queue>,
     events: EventsSender,
     query_validator: Arc<IsQueryAllowedBoxed>,
+    #[allow(dead_code)] // False positive
```

Comment:
```suggestion
    #[allow(dead_code)] // Buffer the feature differences
```

BTW why we keep `telemetry` a feature? Currently most of integration tests depend on the status endpoint via `wait_for_genesis_committed()`

Reply:
It might be useful to remove telemetry under some circumstances. 

As for our client tests... it's very likely that we'll need to replace them with proper integration tests with process isolation. They stay in the codebase for now because we have more important things to worry about.